### PR TITLE
Create temperaturesensor_AHT10.

### PR DESCRIPTION
**Pull request by jpa-welten**
support for a new AHT10 sensor
 * AHT Sensor is connected to I2C on ESP8266 devices --> SCL D1 / SDA D2 
 * To begin, install the necessary library, specifically the Adafruit “AHTX0” package 
 * (you might also get warnings for other libraries missing, likely “Adafruit IO“, and “Adafruit Unified Sensors“: install as needed.)
 * - see https://github.com/adafruit/Adafruit_AHTX0
 * - see https://github.com/adafruit/Adafruit_Sensor
 * - see https://github.com/adafruit/Adafruit_IO_Arduino

### CODE REVIEW

File: examples/temperaturesensor/temperaturesensor_AHT10. [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+#include "SinricProTemperaturesensor.h"
+#include <Adafruit_AHTX0.h>
+Adafruit_AHTX0 aht;
+
+
+#define WIFI_SSID         "JW.ORG"    
+#define WIFI_PASS         "47988414805"
+#define APP_KEY           "1f14d03d-e86f-4d0c-9d78-8379c9b9c394"      // Should look like "de0bxxxx-1x3x-4x3x-ax2x-5dabxxxxxxxx"
+#define APP_SECRET        "33130a8d-cd39-4868-a0fa-cbc82a86ffb6-fb2ac39f-d511-42a2-8f60-17542e1e6b9f"   // Should look like "5f36xxxx-x3x7-4x3x-xexe-e86724a9xxxx-4c4axxxx-3x3x-x5xe-x9x3-333d65xxxxxx"
+#define TEMP_SENSOR_ID    "60faf5acaf7d763d5b0d629e"    // Should look like "5dc1564130xxxxxxxxxxxxxx"
```
Comment (sivar2311) [inline examples/temperaturesensor/temperaturesensor_AHT10.:52]: Please remove your credentials
```C++
#define WIFI_SSID         ""
#define WIFI_PASS         ""
#define APP_KEY           ""    // Should look like "de0bxxxx-1x3x-4x3x-ax2x-5dabxxxxxxxx"
#define APP_SECRET        ""    // Should look like "5f36xxxx-x3x7-4x3x-xexe-e86724a9xxxx-4c4axxxx-3x3x-x5xe-x9x3-333d65xxxxxx"
#define TEMP_SENSOR_ID    ""    // Should look like "5dc1564130xxxxxxxxxxxxxx"
```

File: examples/temperaturesensor/temperaturesensor_AHT10. [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+void handleTemperaturesensor() {
+  if (deviceIsOn == false) return; // device is off...do nothing
+
+  unsigned long actualMillis = millis();
+  if (actualMillis - lastEvent < EVENT_WAIT_TIME) return; //only check every EVENT_WAIT_TIME milliseconds
+
+           sensors_event_t hum, temp;
+           aht.getEvent(&hum, &temp);
+           float humidity = hum.relative_humidity;
+           float temperature = temp.temperature;
```
Comment (sivar2311) [inline examples/temperaturesensor/temperaturesensor_AHT10.:99]: Please fix indention (2 spaces)
```C++
  sensors_event_t hum, temp;
  aht.getEvent(&hum, &temp);
  float humidity = hum.relative_humidity;
  float temperature = temp.temperature; 
```

**Review by sivar2311 — CHANGES_REQUESTED** [review]
Hi and thanks again for the new example.
Please see my change requests.
Additionally, please create a separate directory for each example. 
Name the directory exactly the same as the example itself:
`\temperaturesensor\AHT10\AHT10.ino`

**Review by jpa-welten — COMMENTED** [review]
removed my credentials

**Pull request merged**